www/src/views/starter-library/filtered-starters.js

```diff
@@ -81,6 +94,22 @@ export default class FilteredStarterLibrary extends Component {
             {(filters.size > 0 || urlState.s.length > 0) && ( // search is a filter too https://gatsbyjs.slack.com/archives/CB4V648ET/p1529224551000008
               <ResetFilters onClick={resetFilters} />
             )}
+            <LHSFilter
+              heading="Gatsby Version"
+              data={Array.from(
+                count(
+                  starters.map(
+                    ({ node }) =>
+                      node.fields &&
+                      node.fields.starterShowcase.gatsbyMajorVersion.map(
+                        str => str[1]
```

Comment:
Not a blocking change, but this could potentially be more readable with destructuring:

```js
/* ... */.map(([ _, version ]) => version)
```